web/src/stream_dropdown_list.js

```diff
@@ -0,0 +1,146 @@
+import $ from "jquery";
+import {delegate} from "tippy.js";
```

Comment:
I kinda don't like having a function called just `delegate` rather than `tippy.delegate`, which makes it much clearer without checking the imports what it is.

Reply:
Done.